policy/modules/contrib/rhsmcertd.te

```diff
@@ -47,6 +47,8 @@ dontaudit rhsmcertd_t self:process setfscreate;
 
 allow rhsmcertd_t self:fifo_file rw_fifo_file_perms;
 allow rhsmcertd_t self:unix_stream_socket create_stream_socket_perms;
+allow rhsmcertd_t rhsmcertd_var_run_t:sock_file create;
+allow rhsmcertd_t rhsmcertd_var_run_t:sock_file { setattr unlink };
```

Comment:
I created this patch in two steps at it is described in the issue. Could it be simplified to: `allow rhsmcertd_t rhsmcertd_var_run_t:sock_file { create setattr unlink };`?

Reply:
Yes, the result is the same. A different approach is to change it to manage_sock_file_perms or even using a pattern (see line 72): are you sure no other permission will be needed in the future? 

https://github.com/fedora-selinux/selinux-policy/blob/rawhide/policy/support/obj_perm_sets.spt#L236
https://github.com/fedora-selinux/selinux-policy/blob/rawhide/policy/support/file_patterns.spt#L409

Reply:
I actually think this is a bug introduced by the commiter of 5b4cfb2d6dee9906ead5020303050518da3342c5

Reply:
We need only this ATM. We don't have plan to extend this part of D-Bus API. I will update my PR tomorrow.

Reply:
@zpytela Hi, updated PR as you suggested.

Reply:
LGTM
What I meant with my previous comments is that if I had noticed that, I'd have included manage_sock_files_pattern() in the https://github.com/fedora-selinux/selinux-policy/commit/5b4cfb2d6dee9906ead5020303050518da3342c5 commit.